lib/modules/manager/cargo/extract.spec.ts

```diff
@@ -513,5 +502,84 @@ replace-with = "mcorbin"
       expect(res?.deps).toHaveLength(1);
       expect(res?.deps[0].packageName).toBe('boolector');
     });
+
+    it('extracts locked versions', async () => {
+      const cargolock = Fixtures.get('lockfile-update/Cargo.3.lock');
+      mockReadLocalFile({ 'Cargo.lock': cargolock });
+      fs.findLocalSiblingOrParent.mockResolvedValueOnce('Cargo.lock');
+
+      const cargotoml = codeBlock`[package]
+name = "test"
+version = "0.1.0"
+edition = "2021"
+[dependencies]
+syn = "2.0"`;
```

Comment:
```suggestion
      const cargotoml = codeBlock`
        [package]
        name = "test"
        version = "0.1.0"
        edition = "2021"
        [dependencies]
        syn = "2.0"
        `;
```
`codeBlock` will drop leading whitespace for you 😉